R/ResultsDataModel.R

```diff
@@ -531,6 +535,16 @@ deleteAllRecordsForDatabaseId <- function(connection,
                                           schema,
                                           tableName,
                                           databaseId) {
+  tableExists <-
```

Comment:
There are no guarantees this will work and it doesn't seem related to the PR

Reply:
I can take that off - thank you.

ofcourse if there is no table - the sql will fail. we just have to make sure that deleteAllRecordsForDatabaseId is not used without first knowing there is such a table.